### PR TITLE
aichat: minor ui tweaks

### PR DESCRIPTION
**Pull request by nullhook**
Resolves https://github.com/brave/brave-browser/issues/35177

- input has shadows, new padding, and radius
- conversation turn doesn't have color banding anymore
- scrollbar has a new style

## Submitter Checklist:

- [ ] I confirm that no [security/privacy review is needed](https://github.com/brave/brave-browser/wiki/Security-reviews) and no other type of reviews are needed, or that I have [requested](https://github.com/brave/reviews/issues/new/choose) them
- [ ] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [ ] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally:
  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

**Review by petemill — COMMENTED** [review]
The _blur_ seems a bit ...much... e.g. see example on Storybook when scrolling past code block, the area much further to the side of the code block, below the text box, gets black. It actually clips as well. It's also a little distracting due to the flashing effect you get with blurs scrolling past a small area. So perhaps we should 1) fix the clipping and 2) decrease the blur amount? Or just remove the blur (I tried in devtools and think it looks ok, but up to you) cc @aguscruiz 

image


As for the scrollbar, looks good, but is it now meant to be permanently visible? (I'm ok if so, just want to confirm). Also, did you check it looks ok on Windows?

**Comment by nullhook** [comment]
image

we've determined that the backdrop blur doesn't adapt well to light/dark mode, particularly with certain background content. we've changed it to a solid color while still allowing enough space for the custom scrollbar thumb. this prevents the scrollbar from painting behind the layer.

regarding the custom scrollbar that doesn't respect the OS setting of 'auto' or 'always,' i think it's acceptable. cc @aguscruiz to confirm

**Comment by aguscruiz** [comment]
I'm good with that behavior for the scrollbar

File: components/ai_chat/resources/page/components/main/style.module.scss [hunk 1]
```diff
@@ -12,18 +12,35 @@
...
+  padding: 0 8px;
+  position: absolute;
+  z-index: 3;
+  bottom: 0;
+  left: 0;
+}
+
+.inputPadBottom {
+  background: var(--leo-color-container-background);
+  padding-bottom: 8px;
```
Comment (petemill) [inline components/ai_chat/resources/page/components/main/style.module.scss:28]: I'm not sure why we have these changes anymore in this PR if it's now just a solid color like on master currently? Looks like the only change needed now is changing `padding: 16px` to `padding: 0 16px 16px 16px`?

Comment (aguscruiz) [reply components/ai_chat/resources/page/components/main/style.module.scss:28]: Adding here, the padding changed, no longer 16px but now its `0 8px 8px 8px `

Comment (nullhook) [reply components/ai_chat/resources/page/components/main/style.module.scss]: the subtle distinction in having the input box absolutely positioned is that the scrollbar terminates at its bottom, not the top. however, i don't see any user benefit. in fact, i removed the floating input issue from this PR, which is now re-scoped to include only minor UI changes, as updated in the description

**Review by petemill — APPROVED** [review]

**Pull request merged**